test/isPositiveZero.js

```diff
@@ -0,0 +1,38 @@
+import { assert } from 'chai';
+import * as R from 'ramda';
+
+import * as RA from '../src';
+
+describe('isPositiveZero', function() {
+  context('given a positive zero (+0)', function() {
+    specify('should return true', function() {
+      assert.isTrue(RA.isPositiveZero(+0));
```

Comment:
Last thing: we are missing test for implicit positive zero
assert.isTrue(RA.isPositiveZero(0));

Reply:
I think I did something wrong with Git console when sending this update. So, when I made first edit of the PR, I edited files, then issued `git add -A` in isPositiveZero branch, then `git commit -m "something"`, then `git rebase master -i`, then `git push -f origin isPositiveZero` and everything looks right. Then I tried to make the same and that "git" file appeared (I deleted it here). I think it's because I was in wrong branch or something like this (I don't fully understand the rebase command). How should I make a commit after the first editing and rebasing?

Reply:
I think the issue is resolved, but any additional info on the workflow is highly appreciated.

Reply:
OK, thanks!